README.md

```diff
@@ -1,35 +1,42 @@
-# OpenPower Firmware Build Environment
+# OpenPOWER Firmware Build Environment
```

Comment:
Yeah, seems like most of the IBM literature now has POWER for processors and Power for systems.  So OpenPOWER would make sense...